src/stats/stats_printer.cpp

```diff
@@ -103,6 +121,14 @@ void print_full_stats(socket_stats_t* p_si_stats, mc_grp_info_t* p_mc_grp_info,
 	}
 	fprintf(filename, "\n");
 
+	//
+	// Ring Allocation Logic information
+	//
+	fprintf(filename, "- RX: ring alloc logic = %s, ring user ID = %lu\n",
+			ring_logic_str(p_si_stats->ring_alloc_logic_rx), p_si_stats->ring_user_id_rx);
+	fprintf(filename, "- TX: ring alloc logic = %s, ring user ID = %lu\n",
+			ring_logic_str(p_si_stats->ring_alloc_logic_tx), p_si_stats->ring_user_id_tx);
+
```

Comment:
ring_id is not necessarily coming from the user setting. It's calculated according to logic and current behavior of the application (if it's per thread than the thread number, if it's core than core id). Sometimes it can also be 0 so why not show it?

Reply:
But UDP socket inculde more than a single ring. also tcp listen socket which bounded to 0.0.0.0.
we need to talk about it

Reply:
Please move this section to be printed before socket statistics

Reply:
Done